File: Yogscart Alpha/Assets/Scripts/Game Managers/TimeManager.js
#pragma strict

static function TimerToString(time : float)
{
	var returnString : String = "";
	var timeInt : int = time;
	var milliSeconds : float = (time - timeInt) * 1000f;
	
	returnString = (timeInt/60).ToString("00") + ":" + (timeInt%60).ToString("00") + ":" + milliSeconds.ToString("000");

	return returnString;

}

//add string to float function
static function Parse(time : String) : float
{
	var splitUp : String[] = time.Split(":"[0]);
	
	var returnTime : float = (float.Parse(splitUp[0]) * 60f) + (float.Parse(splitUp[1])) + (float.Parse(splitUp[2])/1000f);
	return returnTime;
	
}

File: Yogscart C#/Assets/Scripts/Game Data/TimeManager.cs
using UnityEngine;
using System.Collections;

public class TimeManager
{
    public static string ToString(float time)
    {
        int timeInt = (int)time;
        float milliSeconds = (time - (float)timeInt) * 1000f;

        return(timeInt / 60).ToString("00") + ":" + (timeInt % 60).ToString("00") + ":" + milliSeconds.ToString("000");
    }

    public static float Parse(string time)
    {
        string[] splitup = time.Split(":"[0]);

        return ((float.Parse(splitup[0]) * 60f) + (float.Parse(splitup[1])) + (float.Parse(splitup[2]) / 1000f));
    }

}
